### PR TITLE
Feat/tags

### PR DESCRIPTION
**Pull request by srishti-maurya**
## Added 
- filtering (date and priority)
- priority

### CODE REVIEW

File: src/frontend/components/FilterList.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import React from "react";
```
Comment (BelideNikhil) [inline src/frontend/components/FilterList.jsx:1]: You can remove this

File: src/frontend/components/LabelList.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useNotes } from "../context";
+import { SingleNoteCard } from "./SingleNoteCard";
+
+function LabelList() {
+  const { state, usedTags } = useNotes();
+  let uniqueTags = usedTags.filter((item, i, ar) => ar.indexOf(item) === i);
```
Comment (BelideNikhil) [inline src/frontend/components/LabelList.jsx:7]: naming convention can be better !

File: src/frontend/components/PriorityList.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { useNotes } from "../context";
+
+export const priorityList = [{ low: "1" }, { medium: "2" }, { high: "3" }];
+
+export function PriorityList({ item }) {
+  const { state, setUserInput, editNote } = useNotes();
+  const matchedNote = state.notesList.find((ele) => ele._id === item._id);
+  console.log(state.notesList);
```
Comment (BelideNikhil) [inline src/frontend/components/PriorityList.jsx:9]: remove console log here

**Review by BelideNikhil — APPROVED** [review]
Try using relative units in CSS
Rest is good!

GTG!

**Review by msdeshmukh009 — APPROVED** [review]
The code looks fine. Just consider changes suggested by @BelideNikhil . Nice work!!

**Review by himadri2110 — APPROVED** [review]
Apart from the changes suggested by @BelideNikhil, everything looks good.

**Pull request merged**